serf/serf.go

```diff
@@ -781,15 +781,33 @@ func (s *Serf) Members() []Member {
 	return members
 }
 
-// RemoveFailedNode forcibly removes a failed node from the cluster
+//RemoveFailedNode is a backwards compatabile form
+// of ForceLeave
+func (s *Serf) RemoveFailedNode(node string) error {
+	if err := s.ForceLeave(node, false); err != nil {
```

Comment:
Since it's just a straight method wrap, you can do this in one line:
```
return s.ForceLeave(node, false)
```
and similar for RemoveFailedNodePrune